cve_bin_tool/cli.py

```diff
@@ -272,18 +272,18 @@ def main(argv=None):
     output_group.add_argument(
         "--metrics",
         action="store_true",
-        help="check for metrics (e.g., EPSS) from found cves",
+        help="check for metrics (e.g., EPSS) from found cves. `--metrics` gets enabled automatically when  either `--epss-percentile` or `--epss-probability` is set",
```

Comment:
```suggestion
        help="check for metrics (e.g., EPSS) from found cves",
```

The help text here is going to be displayed in the console, and it's pretty common for people to have that set to around 80 character width.   I'm going to suggest we just leave this help text as is and only update the other two rather than trying to squash it down.